src/custom_node_interface.h

```diff
@@ -52,7 +52,16 @@ struct CustomNodeParam {
 extern "C" {
 #endif
 
+// custom node library initialize enables creation of resources to be reused between predictions
```

Comment:
If those comments are here to stay, I would capitalize sentences and use doxygen comment notation